### PR TITLE
Add --pacakges options

### PR DESCRIPTION
**Pull request by maldo**
Following the work done here https://github.com/raimon49/pip-licenses/pull/19, I tried to solve the https://github.com/raimon49/pip-licenses/issues/18, following a similar approach

### CODE REVIEW

File: piplicenses.py [hunk 1]
```diff
@@ -753,7 +758,12 @@ def create_parser():
         nargs='+', metavar='PKG',
         default=[],
         help='ignore package name in dumped list')
-
+    common_options.add_argument(
+        '-p', '--packages',
+        action='store', type=str,
+        nargs='+', metavar='PKG',
+        default=[],
+        help='package name in dumped list')
```
Comment (raimon49) [inline piplicenses.py]: This help message here looks better with the text suggested in #19.

```python
help='only include selected packages in output'
```

**Review by raimon49 — APPROVED** [review]
@maldo Thanks for your contribution!

I will take this P-R and upload the next release 3.4.0 to PyPI.

**Comment by raimon49** [comment]
I've shipped it.
https://pypi.org/project/pip-licenses/3.4.0/

**Comment by maldo** [comment]
> @maldo Thanks for your contribution!
> 
> I will take this P-R and upload the next release 3.4.0 to PyPI.

Thank you!

**Pull request merged**